rmc-docs/src/limitations.md

```diff
@@ -0,0 +1,432 @@
+copysignf64 | Yes | |
+cosf32 | Yes | |
+cosf64 | Yes | |
+ctlz | Yes | |
+ctlz_nonzero | Yes | |
+ctpop | Yes | |
+cttz | Yes | |
+cttz_nonzero | Yes | |
+discriminant_value | Yes | |
+drop_in_place | No | |
+exact_div | Yes | |
+exp2f32 | Yes | |
+exp2f64 | Yes | |
+expf32 | Yes | |
+expf64 | Yes | |
+fabsf32 | Yes | |
+fabsf64 | Yes | |
+fadd_fast | No | |
+fdiv_fast | No | |
+float_to_int_unchecked | No | |
+floorf32 | Yes | |
+floorf64 | Yes | |
+fmaf32 | Yes | |
+fmaf64 | Yes | |
+fmul_fast | No | |
+forget | Partial | Generates `SKIP` statement |
+frem_fast | No | |
+fsub_fast | No | |
+likely | Yes | |
+log2f32 | Yes | |
+log2f64 | Yes | |
+log10f32 | Yes | |
+log10f64 | Yes | |
+logf32 | Yes | |
+logf64 | Yes | |
+maxnumf32 | Yes | |
+maxnumf64 | Yes | |
+min_align_of | Yes | |
+min_align_of_val | Yes | |
+minnumf32 | Yes | |
+minnumf64 | Yes | |
+move_val_init | No | |
+mul_with_overflow | Yes | |
+nearbyintf32 | Yes | |
+nearbyintf64 | Yes | |
+needs_drop | Yes | |
+nontemporal_store | No | |
+offset | Partial | Missing undefined behavior checks |
+powf32 | Yes | |
+powf64 | Yes | |
+powif32 | Yes | |
+powif64 | Yes | |
+pref_align_of | Yes | |
+prefetch_read_data | No | |
+prefetch_read_instruction | No | |
+prefetch_write_data | No | |
+prefetch_write_instruction | No | |
+ptr_guaranteed_eq | Partial | |
+ptr_guaranteed_ne | Partial | |
+ptr_offset_from | Partial | Missing undefined behavior checks |
+raw_eq | Partial | Missing undefined behavior checks |
+rintf32 | Yes | |
+rintf64 | Yes | |
+rotate_left | Yes | |
+rotate_right | Yes | |
+roundf32 | Yes | |
+roundf64 | Yes | |
+rustc_peek | No | |
+saturating_add | Yes | |
+saturating_sub | Yes | |
+sinf32 | Yes | |
+sinf64 | Yes | |
+size_of | Yes | |
+size_of_val | Yes | |
+sqrtf32 | Yes | |
+sqrtf64 | Yes | |
+sub_with_overflow | Yes | |
+transmute | Yes | |
+truncf32 | Yes | |
+truncf64 | Yes | |
+try | No | |
+type_id | Yes | |
+type_name | Yes | |
+unaligned_volatile_load | Partial | See [Notes - Concurrency](#concurrency) |
+unaligned_volatile_store | No | See [Notes - Concurrency](#concurrency) |
+unchecked_add | Yes | |
+unchecked_div | Yes | |
+unchecked_mul | Yes | |
+unchecked_rem | Yes | |
+unchecked_shl | Yes | |
+unchecked_shr | Yes | |
+unchecked_sub | Yes | |
+unlikely | Yes | |
+unreachable | Yes | |
+variant_count | No | |
+volatile_copy_memory | Partial | See [Notes - Concurrency](#concurrency) |
+volatile_copy_nonoverlapping_memory | Partial | See [Notes - Concurrency](#concurrency) |
+volatile_load | Partial | See [Notes - Concurrency](#concurrency) |
+volatile_set_memory | No | See [Notes - Concurrency](#concurrency) |
+volatile_store | No | See [Notes - Concurrency](#concurrency) |
+wrapping_add | Yes | |
+wrapping_mul | Yes | |
+wrapping_sub | Yes | |
+write_bytes | Yes | |
+
+#### Atomics
+
+All atomic intrinsics are compiled as an atomic block where the operation is
+performed. But as noted in [Notes - Concurrency](#concurrency), CBMC
```

Comment:
Should we replace the two occurrences of CBMC in this section by RMC?

Reply:
Yes, done.